tex/alg-geom/affine-var.tex

```diff
@@ -289,6 +289,43 @@ \section{Pictures of varieties in $\Aff^1$}
 	the intersection $I \cap J$ is radical.
 \end{remark}
 
+As another easy result concerning the relation between the ideal and variety, we
+have:
+\begin{proposition}
+	[$\VV(-)$ is inclusion reversing]
+	If $I \subseteq J$ then $\VV(I) \supseteq \VV(J)$.
+	Thus $\VV(-)$ is \emph{inclusion-reversing}.
+\end{proposition}
+\begin{ques}
+	Verify this.
+\end{ques}
+Thus, bigger ideals correspond to smaller varieties.
+
+These results will be used a lot throughout the chapter, so it would be useful
+for you to be comfortable with the inclusion-reversing nature of $\VV$.
```

Comment:
well, at least useful for me...? I can't really understand anything if it take me 30 second every time I need to figure out which direction the inclusion goes.